migrations/20241220213859-delete-old-fields-in-cooperatives-and-add-new-ones.js

```
@@ -117,6 +118,23 @@ module.exports = {
       }
     )
   },
-  down() {},
+
+  async down(db) {
+    const backupExists = await db.listCollections({ name: 'cooperation_backup' }).hasNext()
+    if (!backupExists) {
+      console.error("Резервная копия 'cooperation_backup' не найдена. Откат невозможен.")
```

Please write it in English

---

I thought about it, and in this error case, replace the `return` with `throw new Error()` than migration process fails explicitly and prevents data loss

---

fixed